### PR TITLE
Better ERG

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

ERG participation is now a game preference.

If enabled, you get TP'd to the thunderdome on round end.

Otherwise, you get a permanent pacify brain trauma.

## Why It's Good For The Game

Prevents gamering those who don't want to be gamered.

## Changelog
:cl:
add: End round grief toggle button when the round ends.
add: End round grief is now a game preference.
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
Hey that's a great idea. But what if it was a button you could press after round end? The TP to Thunderdome that is. Depending on the day and the shift I may or may not want to go to EORG and flipping my preference just before it happens I think it's a bit much

**Comment by BobJoga** [comment]
> Hey that's a great idea. But what if it was a button you could press after round end? The TP to Thunderdome that is. Depending on the day and the shift I may or may not want to go to EORG and flipping my preference just before it happens I think it's a bit much

Can do.

**Comment by CoreFlare** [comment]
Oh hell yeah.

**Comment by IrkallaEpsilon** [comment]
Bob doing gods work again, thank you :>

**Comment by Investigator77** [comment]
Chad!

**Comment by ShadeAware** [comment]
Excellent work

**Comment by KathrinBailey** [comment]
Did you delete the camera and flasher button or just change their alpha to 0 or something?

**Comment by BobJoga** [comment]
> -edit
> might be worth making the flasher and camera alpha 0 instead of outright removing them since it breaks entertainment monitors.

i didn't even touch them...?

**Comment by BobJoga** [comment]
WAIT FUCK I HATE BYOND

**Comment by BobJoga** [comment]
@KathrinBailey  i fixed them 👍

**Comment by BlackMajor** [comment]
Yer losing out on a lot of what makes ss13 what it is by tryin' to disincline people from EORG shenanagins. What's one minute of furry violence got people talking enough for ye' to want to make a PR like this.

Anywho, peanut aside. Pacify doesn't prevent EORG.
You can still open gas canisters of burn mixes.
You can still BoH bomb.
etc.

**Comment by BobJoga** [comment]
> Yer losing out on a part of ss13's culture by tryin' to disincline people from EORG shenanagins. What's one minute of furry violence got people talking enough for ye' to want to make a PR like this.
> 
> Peanut aside, pacify doesn't prevent EORG.
> You can still open gas canisters of burn mixes.
> You can still BoH bomb.
> Silicons can't be pacified.
> Hell, pretty sure you can even open toxins bombs.
> etc.

this prevents most of it, and some people don't want to deal with bullshit. assuming you're not a griefer you wil go to the thunderdome instead, i don't expect this to be a fix all. so please stop peanutting and go back to citadel.

**Comment by BlackMajor** [comment]
> this prevents most of it, and some people don't want to deal with bullshit. assuming you're not a griefer you wil go to the thunderdome instead, i don't expect this to be a fix all. so please stop peanutting and go back to citadel.

Don't dismiss valid criticism man. Aint a full peanut without chimping. The things I listed are some of the more common things that happen during EORG, you're still going to get round end griffon. The most viable solution, if you really want to do that, is making many of the offending objects stop processing at round end. But that's added work for a feature that's only relevant for one minute of a round.

Basically, I just think this isn't something worth bothering with y'know.

**Comment by BobJoga** [comment]
> > this prevents most of it, and some people don't want to deal with bullshit. assuming you're not a griefer you wil go to the thunderdome instead, i don't expect this to be a fix all. so please stop peanutting and go back to citadel.
> 
> Don't dismiss valid criticism man. Aint a full peanut without chimping. The things I listed are some of the more common things that happen during EORG, you're still going to get round end griffon. The most viable solution, if you really want to do that, is making many of the offending objects stop processing at round end. But that's added work for a feature that's only relevant for one minute of a round.
> 
> Basically, I just think this isn't something worth bothering with y'know.

Yes, thank you, this is valid criticism, but it's about the server culture. Some people do not want to deal with EORG, and i'm trying to make it easier for that to be respected. Of course, some people will inevitably shit up centcomm and the shuttle, but i'm just doing what i can and if necessary staff can hand out small punishments. This isn't a fix all by any means, it is just a convenience.

**Comment by BlackMajor** [comment]
> Yes, thank you, this is valid criticism, but it's about the server culture. Some people do not want to deal with EORG, and i'm trying to make it easier for that to be respected. Of course, some people will inevitably shit up centcomm and the shuttle, but i'm just doing what i can and if necessary staff can hand out small punishments. This isn't a fix all by any means, it is just a convenience.

Thank you for your understanding. I bare concerns on handing out punishments for bypassing the EORG restriction though, it reeks of slippery slope, I wouldn't encourage that. But yeah, to each owns server culture.

**Comment by BobJoga** [comment]
> Thank you for your understanding. I bare concerns on handing out punishments for bypassing the EORG restriction though, it reeks of slippery slope, I wouldn't encourage that. But yeah, to each owns server culture.

I know it does. Don't get me wrong, i want very minor punishments, not excessively fucking people over for doing a funny in like 1 minute of a round. I try to prevent policies at every corner, but i feel this is really the best solution to EORG on our server.

**Comment by BobJoga** [comment]
> EDIT: Hey wouldn't it be WAY EASIER if you just made it so people who had EORG participation turned off end up being teleported away instead of the other way around?

I mean tbh yes, i guess i can change that, but for the moment i'm kinda tired so i'll see if i'll do that later maybe. I don't know.

**Comment by BlackMajor** [comment]
> I mean tbh yes, i guess i can change that, but for the moment i'm kinda tired so i'll see if i'll do that later maybe. I don't know.

Good luck with that one then, I'd recommend doing it. Does away all of my concerns in one fell swoop.

**Comment by BobJoga** [comment]
> Is this finished?

yes
